code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -426,21 +418,20 @@
 	playsound(loc, 'sound/machines/ping.ogg', 30, FALSE)
 
 	var/obj/item/organ/internal/cyberimp/bci/bci_organ = carbon_occupant.getorgan(/obj/item/organ/internal/cyberimp/bci)
-	var/obj/item/organ/internal/cyberimp/bci/bci_to_implant_resolved = bci_to_implant?.resolve()
 
 	if (bci_organ)
 		bci_organ.Remove(carbon_occupant)
 
-		if (isnull(bci_to_implant_resolved))
+		if (!bci_to_implant)
```

Comment:
Changing these from `isnull` to `!x` is unnecessary and I would rather you kept what it was.

Reply:
Huh, why?

Reply:
Not opposed, though. Will do.

Reply:
Cuz I wrote it and I like the explicitness

Reply:
Well if that's the reason I disagree now. A lot of other files use !x and the only thing better than perfect is standardized.

Plus !x is more common in all languages that support it afaik and is more compact.

Reply:
Okay it's necessary to change it back as it's one of the things we care that we don't care about, like `if (x)` vs `if(x)`

Reply:
I'll do it for this, no problem. However if I make a new feature I'm still going to use the !x method. It's engrained into my coding habits far too deeply.

Reply:
I don't use !x where I could've used isnull

Reply:
Yet somehow there's !occupant mixed in there a whole bunch.